src/components/Select/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChangeEvent, Component } from 'react';
+import { SelectProps } from '../../types/component';
+
+class Select extends Component<SelectProps> {
```

Comment:
정말로 범용적으로 쓸거라면, html 의 select요소와 인터페이스를 똑같이 맞춰주는게 좋습니다.
(왜 좋을까요? 는 질문으로 남겨둡니다..)

```suggestion
class Select extends Component<HTMLAttributes<HTMLSelectElement>> {
```

Reply:
`HTMLAttributes<HTMLSelectElement>` 이런 게 있는지 몰랐어요!! 알려주셔서 감사합니다 👍